picard/ui/mainwindow.py

```diff
@@ -784,17 +784,21 @@ def analyze(self):
                 return
         return self.tagger.analyze(self.selected_objects)
 
+    def _openUrl(self,url):
+        if url.startswith("\\\\") or url.startswith("//"):
```

Comment:
Please add a comment about the reason of this, it is a workaround for a Qt bug, so it may be removed at some point.